kiwi/defaults.py

```diff
@@ -59,7 +57,6 @@ def __init__(self):
         }
         self.profile_key_list = [
             'kiwi_align',
-            'kiwi_startsector',
```

Comment:
Very concerned about this change. The start sector must be in the profile so that the initrd code can access it for its repartition tasks. The method to_profile() in the Defaults space makes use of this values and makes sure they all land in the .profile file. Deleting it from this list has very intrusive consequences. I saw you were aware of this problem and addressed in the profile code but the reason why I did not put those into extra methods was that the values belongs together

I understand that an extra method for the start sector is nice but I would not go that route for start_sector, alignment and sectorsize. They all belong together and should not be separated.

Please keep the list intact, it will also make the amount of changes smaller

Reply:
I understand you point that they all belong together these values. I'll work to re-add the kiwi_startsector into the list, but then I guess I will need to modify somehow the `Profile.add` method and/or keep the `Profile._type_to_profile()` as is in this PR, otherwise the profile file includes always `kiwi_startsector='2048'` as a fixed value even the disk_start_sector was set to something different. In fact I included this change after testing and noticing the profile file was not including the correct value. I'll try to address it and see if I can find a convincing solution.